src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/BookingService.kt

```diff
@@ -1335,6 +1333,7 @@ class BookingService(
     cancelledAt: LocalDate,
     reasonId: UUID,
     notes: String?,
+    user: UserEntity?,
```

Comment:
I've been making changes in the same area so the rebase is going to cause you some headaches (sorry)

But - i've changed it so the controller directly calls the Cas1 or Cas3 endpoins because the argument lists were significantly diverging. I _think_ this means you can make user non-nullable if createCas3Cancellation is always going to be called via a controller?

Reply:
done